test/libsolidity/smtCheckerTests/crypto/crypto_functions_same_string_literal.sol

```diff
@@ -0,0 +1,23 @@
+contract C {
+	function c1() public pure {
+		bytes32 k1 = keccak256("1");
+		bytes32 k2 = keccak256("1");
+		assert(k1 == k2);
+	}
+
+        function c2() public pure {
+		bytes32 s1 = sha256("10");
+		bytes32 s2 = sha256("10");
+		assert(s1 == s2);
+	}
+
+        function c3() public pure {
```

Comment:
Pleae add a negative example like
```
 function c4() public pure {
		bytes32 s1 = sha256("110");
		bytes32 s2 = sha256("100");
		assert(s1 == s2); // should fail
	}
```
and similar for array literal case

Reply:
Hmm, so I tried to add a negative example and it added 7 seconds to the runtime of the test.
For that reason I would prefer not adding the negative test, as I don't think it adds much value.
There are other tests that compare `keccak` of different `bytes`, so maybe we don't need the same for string literals, considering the trade-off?

Reply:
ok